framework/src/main/java/org/checkerframework/framework/stub/StubParser.java

```diff
@@ -1504,6 +1516,33 @@ private Object convert(Number number, TypeKind expectedKind) {
         }
     }
 
+    /**
+     * Converts {@code number} to {@code expectedKind} * -1
+     *
+     * @param number Number value to be converted
+     * @param expectedKind converts the Number to {byte, short, int, long, float, double}
```

Comment:
This isn't a comment about the formal parameter.
Maybe you are trying to say,
"the type of the result.  Must be one of {...}."

Reply:
addressed